tests/pytests/integration/modules/test_mac_timezone.py

```diff
@@ -0,0 +1,225 @@
+"""
+Integration tests for mac_timezone
+
+If using parallels, make sure Time sync is turned off. Otherwise, parallels will
+keep changing your date/time settings while the tests are running. To turn off
+Time sync do the following:
+    - Go to actions -> configure
+    - Select options at the top and 'More Options' on the left
+    - Set time to 'Do not sync'
+"""
+
+import datetime
+
+import pytest
+
+pytestmark = [
+    pytest.mark.flaky(max_runs=4),
+    pytest.mark.skip_if_binaries_missing("systemsetup"),
+    pytest.mark.slow_test,
+    pytest.mark.destructive_test,
+    pytest.mark.skip_if_not_root,
+    pytest.mark.skip_unless_on_darwin,
+]
+
+
+@pytest.fixture(scope="function")
+def setup_teardown_vars(salt_call_cli):
```

Comment:
```suggestion
@pytest.fixture(scope="function", autouse=True)
def _setup_teardown_vars(salt_call_cli):
```
?

Since it's not getting used anywhere...